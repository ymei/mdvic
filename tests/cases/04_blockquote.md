> Quote that should wrap into multiple lines to verify the prefix handling.

