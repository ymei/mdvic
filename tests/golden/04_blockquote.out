> Quote that should wrap into multiple l
> ines to verify the prefix handling.

